scrapers/Karups.yml

```diff
@@ -0,0 +1,26 @@
+name: "Karups"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - www.karups.com/video/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $details: //div[@class="content-information"]
+    scene:
+      Title: //h1[@class="page-heading"]/span[@class="title"]/text()
+      Date:
+        selector: $details//span[@class="date"]/span[@class="content"]/text()
+        replace:
+          - regex: (st|nd|rd|th)\,
+            with: ","
+        parseDate: Jan 02, 2006
+      Details:
+        selector: //div[@class="content-information-description"]/p/text()
+      Performers:
+        Name: $details//span[@class="models"]/span[@class="content"]//a/text()
+      Image:
+        selector: //video[@id="player"]/@poster
```

Comment:
in some scenes (eg `https://www.karups.com/video/capri-and-tenaya-29995.html` ) the image is part of a div
```yaml
      Image:
        selector: //video[@id="player"]/@poster|//div[@class="video-poster"]/img/@src
```
seems to work